README.md

```diff
@@ -191,9 +193,9 @@ A few other ENB suggestions are:
 
 ### Requiem
 
-[Requiem](https://www.nexusmods.com/skyrim/mods/19281) is a project with the goal of turning Skyrim into a better roleplaying experience by making game more immersive, semi-realistic and coherent, with the tend to find a compromiss between old school mechanics and more modern approaches.
+[Requiem](https://www.nexusmods.com/skyrim/mods/19281) is a project with the goal of turning Skyrim into a better roleplaying experience by making game more immersive, semi-realistic and coherent, and tends to find a compromise between old school mechanics and more modern approaches.
```

Comment:
I had to guess with "tends" here; the original meaning was not at all clear to me.